src/app.ts

```diff
@@ -579,9 +589,9 @@ container.register(
 );
 
 container.register(
-    'storageHandler',
-    asFunction((accountService, storageService) =>
-        createStorageHandler(accountService, storageService),
+    'imageUploadHandler',
+    asFunction((accountService, imageStorageService) =>
```

Comment:
This can just be `asFunction(createImageUploadHandler)` 😝

Reply:
true! updated in d4c5dabae26cbc8f5cd3e9c880a1e78f4bfb1b50